pegasus/sites.v3/hourofcode.com/public/prizes/index.md

```diff
@@ -17,13 +18,20 @@ Every educator who hosted an Hour of Code for students received 10 GB of Dropbox
 
 ## Prizes for EVERY organizer
 
-**Every** educator who hosted an Hour of Code is eligible to receive **$10 to Amazon.com, iTunes or Windows Store** as a thank-you gift!*
-
-<img style="float:left;" src="/images/fit-130/amazon_giftcards.png"/>
-
-<img style="float:left;" src="/images/fit-130/apple_giftcards.png"/>
-
-<img styel="float:left;" src="/images/fit-130/microsoft_giftcards.png"/>
+**Every** educator who hosts an Hour of Code is eligible to receive **$10 to Amazon.com, iTunes, Windows Store or 10GB of Dropbox space** as a thank-you gift!*
+
+<div class="prizes-logo">
```

Comment:
Definitely like using logos over the fan images, but I would include the images as a haml view.